dev/com.ibm.ws.container.service/src/com/ibm/ws/container/service/annocache/internal/AnnotationsImpl.java

```diff
@@ -735,7 +744,76 @@ protected void addExternalToClassSource() {
         rootClassSource.addClassLoaderClassSource(classLoaderClassSource);
     }
 
-    //
+    /**
+     * Determine if the Jakarta EE Common Annotations APIs are enabled for the classloader
+     * of the current container.
+     *
+     * @return true if the Jakarta EE Common Annotations APIs are enabled, otherwise false.
+     */
+    @Trivial
+    @FFDCIgnore(ClassNotFoundException.class)
+    private boolean jakartaCommonAnnotationAPIs() {
+        if (classLoader == null) {
+            if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+                Tr.debug(tc, "Jakarta Common Annotations not present - ClassLoader not set");
+            }
+            return false;
+        }
+        try {
+            classLoader.loadClass("jakarta.annotation.Resource");
+            if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+                Tr.debug(tc, "Jakarta Common Annotations present");
+            }
+            return true;
+        } catch (ClassNotFoundException e) {
+            if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+                Tr.debug(tc, "Jakarta Common Annotations not present");
+            }
+            return false;
+        }
+    }
+
+    /**
+     * Check for use of the wrong package (javax.annotation instead of jakarta.annotation)
+     * for the Jakarta Common Annotation APIs where the "javax" version of the annotations
+     * were previously included in the JDK. An informational message will be logged to
+     * the console for each annotation type found.
+     *
+     * @param targets the scanned annotation targets for the current container
+     * @param scanPolicies The policies for which to select annotated classes, as bitwise
+     *            OR of scan policy values.
+     */
+    private void checkForWrongPackageCommonAnnotations(AnnotationTargets_Targets targets, int scanPolicies) {
+        if (modName == null || !jakartaCommonAnnotationAPIs()) {
+            return;
+        }
+
+        Set<String> foundClasses = new TreeSet<String>(); // provides natural ordering
+
+        // Look for @Resoruce at class, field, and method level and nested in @Resoruces at class level
```

Comment:
"@Resoruce" - spelled wrong in the comment

Reply:
fixed